src/settings-ui/Settings.UI/ViewModels/DashboardViewModel.cs

```diff
@@ -78,7 +78,7 @@ private void AddDashboardListItem(ModuleType moduleType)
                 IsLocked = gpo == GpoRuleConfigured.Enabled || gpo == GpoRuleConfigured.Disabled,
                 Icon = ModuleHelper.GetModuleTypeFluentIconName(moduleType),
                 EnabledChangedCallback = EnabledChangedOnUI,
-                AccentColor = GetModuleAccentColor(moduleType),
+                AccentColor = ModuleHelper.GetModuleAccentColor(moduleType),
```

Comment:
Why doing this? For me it looks wired that the cards have different colors.

Reply:
This is a first time, someone doesn't like it, I heard only positive feedback about the different colors. Our style expert added this feature and the plan is to keep it.